autoload/pythonimports.vim

```diff
@@ -38,13 +38,18 @@ function! pythonimports#filename2module(filename)
     let root = found_dir
   endif
 
+  let path_sep = "/"
+  if has("win32")
+      let path_sep = "\\"
+  endif
```

Comment:
So now I'm wondering about things like Cygwin/Mingw.  IIRC paths still use `/` there, but what I don't know is whether Vim will `has("win32")` or not.

How about using a regexp that accepts both `/` and `\`, and doing that on all OSes?

Or how about replacing all `\` with `/` before the rest of the manipulations?  E.g. I see that you've missed the `/` in `^site-packages/` removal.

Reply:
Good call, that is done